tests/integration/pages/home.spec.js

```diff
@@ -0,0 +1,17 @@
+import { Selector } from "testcafe";
+import { loginGoogleUserAndSetLocale } from "../helpers/loginByGoogle";
+
+fixture("Home - English")
+  .page(`${process.env.BROWSERSTACK_BASE_URL}/login`)
+  .beforeEach(async (testcafe) => {
+    await loginGoogleUserAndSetLocale();
+  });
+
+test("sees the home page", async (testcafe) => {
+  const noPlioSection = Selector('[data-test="noPlio"]');
```

Comment:
this test failed for me as it used my local DB values and it had plios (whereas this case will be true when there are no plios locally)